src/index.js

```diff
@@ -77,12 +77,31 @@ const createList = () => {
     }
     localStorage.setItem('list', JSON.stringify(empty));
   });
+// clear all
+const clearAll = document.querySelector('.clear');
+clearAll.addEventListener('click', () => {
+  const getting = JSON.parse(localStorage.getItem('list'));
+  const variable = document.querySelectorAll('.change');
+  for (let i = 0; i < variable.length; i += 1) {
+    form.removeChild(variable[i]);
+  }
+  const empty = [];
+  for (let i = 0; i < getting.length; i += 1) {
```

Comment:
- [ ] As suggested by the previous code reviewer and project requirements. I kindly suggest that you make use of the filter method for your "clearAll" function

##  EXAMPLE
```
export default function clearAll(data = getDataLS()) {
  data = data.filter((task) => !task.completed);
  localStorage.setItem('tasks', JSON.stringify(data));
  indexValue();
}
```